puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/infra/updateJobs.groovy

```diff
@@ -17,6 +17,14 @@ pipeline {
             }
         }
 
+        stage('Update ci.theforeman.org jobs') {
+            steps {
+                withCredentials([string(credentialsId: 'theforeman-jenkins', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+                    virtEnv('jjb-venv', "cd ./ci/theforeman.org && jenkins-jobs --conf ./foreman_jenkins.ini --user ${env.USERNAME} --password '${env.PASSWORD}' update --delete-old -r ./jobs")
```

Comment:
```suggestion
                    virtEnv('jjb-venv', "cd ./ci/theforeman.org && jenkins-jobs --conf ./foreman_jenkins.ini --user ${env.USERNAME} --password '${env.PASSWORD}' update --delete-old -r .")
```

the directory layout is not 100% identical between the tfm and the centos jenkins